ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
We could download the script once and use that until it breaks, that would be a step forward. We could also isolate this by running it in a separate (empty) docker container.